pyproject.toml

```diff
@@ -1,2 +1,26 @@
 [tool.black]
 line-length = 120
```

Comment:
If we are removing black, then presumably this config is redundant and can be safely removed?

Reply:
yes 👍